letsencrypt-auto-source/pieces/pipstrap.py

```diff
@@ -4,7 +4,7 @@
 Embed this in your project, and your VCS checkout is all you have to trust. In
 a post-peep era, this lets you claw your way to a hash-checking version of pip,
 with which you can install the rest of your dependencies safely. All it assumes
-is Python 2.6 or better and *some* version of pip already installed. If
+is Python 2.7 or better and *some* version of pip already installed. If
```

Comment:
So this file is taken from https://github.com/erikrose/pipstrap and I'd like to keep it in sync with what's in that repo. We should revert these changes for now. If you'd like to make this change, I'd create a PR against that repo and if it lands we can copy it here, although Erik may want to keep Python 2.6 support in pipstrap.